clients/apps/web/src/components/Settings/OrganizationCustomerPortalSettings.tsx

```diff
@@ -0,0 +1,153 @@
+import { useUpdateOrganization } from '@/hooks/queries'
+import { useAutoSave } from '@/hooks/useAutoSave'
+import { setValidationErrors } from '@/utils/api/errors'
+import { isValidationError, schemas } from '@polar-sh/client'
+import Switch from '@polar-sh/ui/components/atoms/Switch'
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+} from '@polar-sh/ui/components/ui/form'
+import React from 'react'
+import { useForm } from 'react-hook-form'
+import { toast } from '../Toast/use-toast'
+import { SettingsGroup, SettingsGroupItem } from './SettingsGroup'
+
+interface OrganizationCustomerPortalSettingsProps {
+  organization: schemas['Organization']
+}
+
+const OrganizationCustomerPortalSettings: React.FC<
+  OrganizationCustomerPortalSettingsProps
+> = ({ organization }) => {
+  const form = useForm<schemas['OrganizationCustomerPortalSettings']>({
+    defaultValues: {
+      usage: {
+        show: organization.customer_portal_settings?.usage?.show ?? true,
```

Comment:
Once you'll remove `total=False` on the Python schema (see comment above), we'll be able to remove lot of optional chaining operators.